<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main</class>
 <widget class="QWidget" name="main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Helhed Configuration Writer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff/>
   </iconset>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="labWebAddr">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Web Server Addr:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="webAddr"/>
     </item>
     <item>
      <widget class="QCheckBox" name="tls_ssl">
       <property name="text">
        <string>Enable TLS/SSL</string>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="labAppPath">
       <property name="text">
        <string>Application Path:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="appPath"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="labFTPAddr">
         <property name="text">
          <string>FTP Server Addr:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ftpAddr"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>FTP Port (default 21):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ftpPort">
         <property name="inputMask">
          <string>D0000; </string>
         </property>
         <property name="maxLength">
          <number>5</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="labFTPUser">
       <property name="text">
        <string>FTP Username:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ftpUser"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="labFTPPass">
       <property name="text">
        <string>FTP Password:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ftpPass">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="labUpPath">
       <property name="text">
        <string>Upload Path:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ftpUpload"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="stdLabel">
       <property name="text">
        <string>Temp Upload Path:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ftpTempUpload"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Default Style Theme:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="defStyle">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string comment="cleanlooks">Cleanlooks</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string comment="CDE">Unix CDE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string comment="plastique">KDE Plastique</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string comment="winxp">Windows XP</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="fontszLab">
       <property name="text">
        <string>Default Font Size:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="defaultFontSize"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="Write">
       <property name="text">
        <string>Write Config</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="endDlg">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
